### ISSUE TITLE
simple run Error

### ISSUE THREAD

**Issue by DavidDXY** [SteveSandersonMS/WebWindow#112]
when the app load the html url ,throw this error：
System.AccessViolationException:“Attempted to read or write protected memory. This is often an indication that other memory is corrupt.”

**Comment by nmocruz** [comment]
I have this one to.

**Comment by nmocruz** [comment]
@DavidDXY I solved it on the samples just changing the target to x64.

**Comment by DapperDeer** [comment]
I believe that this is being caused by a missing file: `blazor.desktop.js` which is what wwwroot/index.html is trying to call, but there is no js file associated with the sample. I'm not sure if it's supposed to be generated during build, but I do believe this is what's causing the error.

**Comment by igor-lukashenya** [comment]
> I believe that this is being caused by a missing file: `blazor.desktop.js` which is what wwwroot/index.html is trying to call, but there is no js file associated with the sample. I'm not sure if it's supposed to be generated during build, but I do believe this is what's causing the error.
> 
> Edit: Nope, I'm incorrect. Removing the line still comes up with the memory error and its referenced throughout other Blazor pages. I just installed the new Edge and changed the target to x64, still with no luck.

Do yo resolve this issue?

**Comment by DapperDeer** [comment]
> 
> Do yo resolve this issue?

Yes, actually. I fixed this issue by downloading [Edge Insider ](https://www.microsoftedgeinsider.com/en-us/).

**Comment by devbf** [comment]
Had the same issue, solved it by adding [this](https://www.nuget.org/packages/Microsoft.Web.WebView2) SDK via NuGet.
